topics/variant-analysis/tutorials/tb-variant-analysis/tutorial.md

```diff
@@ -123,54 +123,59 @@ While one could examine the quality control report for each set of reads (forwar
->       - *"Single-end or paired-end reads?"*: `Paired End (two separate input files)`
->       - {% icon param-files %} *"Input FASTQ file (R1/first of pair)"*: `004-2_1.fastq.gz`
->       - {% icon param-files %} *"Input FASTQ file (R2/second of pair)"*: `004-2_2.fastq.gz`
->       - *"Select Trimmomatic operation to perform"*
->           - Keep the default value of `Sliding window trimming` but set
->           - *"Average quality required"*: `30`
->       - *"+Insert Trimmomatic Operation"*
->           - *"Select Trimmomatic operation to perform"*: `Drop reads below a specified length (MINLEN)`
->           - *"Minimum length of reads to be kept"*: `20`
->
-> 2. Inspect the output produced by Trimmomatic
+> 1. Use {% tool [fastp](toolshed.g2.bx.psu.edu/repos/iuc/fastp/fastp/0.23.4+galaxy0) %} to clean up the reads and remove the poor quality sections.
+>       - *"Single-end or paired-end reads?"*: `Paired`
+>       - {% icon param-files %} *"Input 1"*: `004-2_1.fastq.gz`
+>       - {% icon param-files %} *"Input 2"*: `004-2_2.fastq.gz`
+>
+>
+> 2. Inspect the output produced by **fastp**
 >
 >    > <question-title></question-title>
->    >
->    > 1. Why are there 4 output read files instead of 2?
+>     >
+>    > 1. Was there a difference in file size of the reads before and after trimming? Is the difference the same between both sets of reads?
 >    >
 >    > > <solution-title></solution-title>
 >    > >
->    > > 1. There are 4 output files: Forwards paired and single reads and reverse paired and single reads. The single reads come about when one read in a pair of reads has failed the quality checks and so is deleted. The other half of the pair may still be good and so it is put into the single reads file for the appropriate direction. While un-paired reads might sometimes be useful, paired reads are more useful because both their sequence and the gap between them (the "insert size") can be used for further analysis. In a typical analysis, only the paired reads are used.
+>    > > 1. After processing by **fastp** the output files are about 80 MB smaller than the inputs. The change in size is similar for both files.
 >    > >
 >    > {: .solution}
 >    {: .question}
 {: .hands_on}
 
-*Note:* We would normally examine our trimmed reads with **FastQC** and **MultiQC** again to see if the quality trimming has been successful, but in this tutorial we will move straight on to save time.
+*Note:* We would normally examine our trimmed reads with **FastQC** and **MultiQC** again to see if the quality trimming has been successful, but in this tutorial, we will move straight on to save time.
+
+The next section, on looking for contamination in our data using Kraken2, takes a long time to run and can be skipped and perhaps
+run later if you prefer.
+
+{% include _includes/cyoa-choices.html option1="Skip-Kraken2" option2="Run-Kraken2" default="Skip-Kraken2" %}
```

Comment:
```suggestion
{% include _includes/cyoa-choices.html option1="Skip Kraken2" option2="Run Kraken2" default="Skip Kraken2" %}
```

spaces will be replaced automatically with - when generating the div classes (and be more readable,)so  no changes needed to divs, this looks good